### ISSUE TITLE
missing unit for ASE on Output

### ISSUE THREAD

**Issue by ggrammel** [Telecominfraproject/oopt-gnpy#201]
**Describe the bug**

python transmission_main_example.py CORONET_Global_Topology.json

output (shortened)
```
Roadm roadm Atlanta
  loss (dB):     20.00
  pch out (dBm): -20.0
Transceiver trx Atlanta
  OSNR ASE (1nm):        17.45
  OSNR ASE (signal bw):  13.37
  SNR total (signal bw): 11.07

Transmission result for input power = 0.00dBm :
Transceiver trx Atlanta
  OSNR ASE (1nm):        17.45
  OSNR ASE (signal bw):  13.37
  SNR total (signal bw): 11.07
```
Unlike all other data, neither ASE nor SNR have units associated

**To Reproduce**
see above

**Expected behavior**
Output should include units

**Screenshots**

**Environment:**
 - OS: MacOS
 - Python Version 3.6.5
 - Anaconda Version [e.g. 3.7]

**Additional context**

**Comment by dutc** [comment]
Let's put together a quick fix for this for v1.1.

I believe both OSNR ASE and SNR total are dB measurements, but we can confirm that on the call:

They are computed as follows:
```
            self.osnr_ase = [lin2db(divide(c.power.signal, c.power.ase))
                            for c in spectral_info.carriers]
            self.snr = [lin2db(divide(c.power.signal, c.power.nli + c.power.ase)) 
                        for c in spectral_info.carriers]
```

**Comment by jeanluc-auge** [comment]
yes these are dB information, we can add it
the important info is about 0.1nm and signal bandwidth

**Comment by EstherLerouzic** [comment]
this has been adressed in PR #216 commit
[https://github.com/Telecominfraproject/oopt-gnpy/pull/216/commits/ca9d634086b6eb70a86c57c667a5f179d9017f02]

**Comment by dutc** [comment]
Fixed in v1.2 branch.
